### PR TITLE
Return links that are still on API to tests

### PR DESCRIPTION
**Pull request by liry**
We still want to test we're able to deserialize it when it's still on API.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-java-model/src/test/resources/project/projects.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
                         "eventstores": "/gdc/projects/defaultEmptyProject/dataload/eventstore/stores",
                         "execute": "/gdc/projects/defaultEmptyProject/execute",
                         "clearCaches": "/gdc/projects/defaultEmptyProject/clearCaches",
+                        "projectFeatureFlags": "/gdc/projects/defaultEmptyProject/projectFeatureFlags",
```
Comment (peter-plochan) [inline gooddata-java-model/src/test/resources/project/projects.json:47]: Shouldn't we just introduce something like `"alreadyDeprecatedLink": "...",` to properly illustrate that we are able to accept deprecated links? 🤷

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-java-model/src/test/resources/project/projects.json (hunk 1 above)
Comment (liry) [reply gooddata-java-model/src/test/resources/project/projects.json:47]: Good idea, added "unknownLink" ;)

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-java-merge-compliance-check/272/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-java-fossa-scanning-zuul-el7/111/ : SUCCESS in 1m 09s (non-voting)

**Pull request merged**